subsys/bluetooth/host/Kconfig

```diff
@@ -338,9 +338,11 @@ if BT_SMP
 config BT_PRIVACY
 	bool "Privacy Feature"
 	help
-	  Enable local Privacy Feature support. This makes it possible
+	  Enable Privacy Feature support. This makes it possible
 	  to use Resolvable Private Addresses (RPAs).
 
+	  Disabling this will remove the capability to resolve private addresses.
```

Comment:
or to create RPAs as well

Reply:
Use the words generate and resolve explicitly to make it clear.